app/controllers/api/auth/stories_controller.rb

```diff
@@ -82,17 +86,13 @@ def search
   def search_params
     sparams = super
     sparams[:fq]['series_id'] = 'NULL' if filters.noseries?
-    sparams[:fq]['published_at'] = search_by_state(filters.state) if filters.state?
+    sparams[:state] = search_by_state(filters.state) if filters.state?
```

Comment:
This is the change to how the ES json is built ... instead of abusing the `:fq` fielded search, this just manually sets up the range/bool filters.